src/app/(views)/managed-courses/[courseId]/page.tsx

```diff
@@ -61,66 +103,223 @@ export default function Course() {
   }
 
   return (
-    <div>
-      <Link href={`/managed-courses/${course.id}/edit`}>Edit</Link> /
-      <button type="button" onClick={() => deleteCourse(course.id)} disabled={deletingCourse}>
-        Delete
-      </button>
-      <p>Course ID: {courseId}</p>
-      <p>Title: {course.title}</p>
-      <p>Description: {course.description}</p>
-      <p>
-        Enrollees:
-        {course.enrollees.map((enrollee, index) => (
-          <span key={enrollee.id}>
-            {enrollee?.name} {index < course.enrollees.length - 1 ? ', ' : ''}
-          </span>
-        ))}
-      </p>
-      <p>
-        Instructors:
-        {course.instructors.map((instructor, index) => (
-          <span key={instructor.id}>
-            {instructor?.name} {index < course.instructors.length - 1 ? ', ' : ''}
-          </span>
-        ))}
-      </p>
-      <table>
-        <thead>
-          <tr>
-            <th colSpan={3}>
-              <Link href={`/managed-courses/${course.id}/modules/create`}>Create Module</Link>
-            </th>
-          </tr>
-          <tr>
-            <th>ID</th>
-            <th>Title</th>
-            <th>Type</th>
-            <th>Actions</th>
-          </tr>
-        </thead>
-        <tbody>
-          {course.modules.map((module) => (
-            <tr key={module.id}>
-              <td>
-                <Link href={`/managed-courses/${course.id}/modules/${module.id}`}>{module.id}</Link>
-              </td>
-              <td>{module.title}</td>
-              <td>{module.type}</td>
-              <td>
-                <Link href={`/managed-courses/${course.id}/modules/${module.id}/edit`}>Edit</Link>
-                <button
-                  type="button"
-                  onClick={() => deleteModule(module.id)}
-                  disabled={deletingModule}
+    // <div>
```

Comment:
Do we need this commented code? if yes, put a comment above explaining why this is still here.